x-pack/plugins/siem/public/components/open_timeline/timelines_table/common_columns.tsx

```diff
@@ -0,0 +1,120 @@
+ * and the full view shown in the `All Timelines` view of the `Timelines` page
+ */
+export const getCommonColumns = ({
+  itemIdToExpandedNotesRowMap,
+  onOpenTimeline,
+  onToggleShowNotes,
+  showExtendedColumnsAndActions,
+}: {
+  onOpenTimeline: OnOpenTimeline;
+  onToggleShowNotes: OnToggleShowNotes;
+  showExtendedColumnsAndActions: boolean;
+  itemIdToExpandedNotesRowMap: Record<string, JSX.Element>;
+}) => [
+  {
+    width: '32px',
+    isExpander: true,
+    render: ({ notes, savedObjectId }: TimelineResult) =>
+      notes != null && notes.length > 0 && savedObjectId != null ? (
+        <ExpandButtonContainer>
+          <EuiButtonIcon
+            data-test-subj="expand-notes"
+            onClick={() =>
+              itemIdToExpandedNotesRowMap[savedObjectId] != null
+                ? onToggleShowNotes(omit(savedObjectId, itemIdToExpandedNotesRowMap))
+                : onToggleShowNotes({
+                    ...itemIdToExpandedNotesRowMap,
+                    [savedObjectId]: <NotePreviews notes={notes} />,
+                  })
+            }
+            aria-label={itemIdToExpandedNotesRowMap[savedObjectId] ? i18n.COLLAPSE : i18n.EXPAND}
+            iconType={itemIdToExpandedNotesRowMap[savedObjectId] ? 'arrowDown' : 'arrowRight'}
+          />
+        </ExpandButtonContainer>
+      ) : null,
+  },
+  {
+    dataType: 'string',
+    field: 'title',
+    name: i18n.TIMELINE_NAME,
+    render: (title: string, timelineResult: TimelineResult) =>
+      timelineResult.savedObjectId != null ? (
+        <EuiLink
+          data-test-subj={`title-${timelineResult.savedObjectId}`}
+          onClick={() =>
+            onOpenTimeline({
+              duplicate: false,
+              timelineId: `${timelineResult.savedObjectId}`,
+            })
+          }
+        >
+          {isUntitled(timelineResult) ? i18n.UNTITLED_TIMELINE : title}
+        </EuiLink>
+      ) : (
+        <div data-test-subj={`title-no-saved-object-id-${timelineResult.title || 'no-title'}`}>
+          {isUntitled(timelineResult) ? i18n.UNTITLED_TIMELINE : title}
+        </div>
+      ),
+    sortable: true,
+  },
+  {
+    dataType: 'string',
+    field: 'description',
+    name: i18n.DESCRIPTION,
+    render: (description: string) => (
+      <span data-test-subj="description">
+        {description != null && description.trim().length > 0 ? description : getEmptyTagValue()}
+      </span>
+    ),
+    sortable: true,
+    width: showExtendedColumnsAndActions ? EXTENDED_COLUMNS_DESCRIPTION_WIDTH : DESCRIPTION_WIDTH,
+  },
+  {
+    dataType: 'date',
+    field: 'updated',
+    name: i18n.LAST_MODIFIED,
+    render: (date: number, timelineResult: TimelineResult) => (
+      <div data-test-subj="updated">
+        {timelineResult.updated != null ? (
+          <FormattedDate fieldName="" value={date} />
+        ) : (
+          getEmptyTagValue()
+        )}
+      </div>
+    ),
+
+    sortable: true,
```

Comment:
I guess you're deciding to keep these items sorted by their alphabet? Looks weird considering all the others in the code base have `render` last and this is the only one that doesn't. I almost missed seeing the `sortable: boolean` here.

Just mentioning, it's the only inconsistent one.